<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerDialog</class>
 <widget class="QDialog" name="FileManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>620</height>
    </rect>
   </property>
   <widget class="QWidget" name="top" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QWidget" name="icon" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>360手机助手</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClose">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>7</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tools" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>850</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QToolButton" name="tbShangceng">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbXinjian">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbShangchuan">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>5</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbFasong">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>5</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbShanchu">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>5</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbFuzhi">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>5</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbZhantie">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>5</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbShuaXin">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>5</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>850</width>
      <height>500</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>851</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>850</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>40</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20个文件夹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>34个文件</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>大小: 12.4G</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>剩余空间: 9.39G</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>备注:除sdcard目录外,其他文件不可修改.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
